pom.xml

```
@@ -85,6 +85,9 @@
     <version.commons-collections4>4.4</version.commons-collections4>
     <version.org.eclipse.jgit>5.11.1.202105131744-r</version.org.eclipse.jgit>
     <version.commons-text>1.9</version.commons-text>
+    <version.dependency-check-core>6.1.6</version.dependency-check-core>
+    <version.slf4j-simple>1.7.30</version.slf4j-simple>
+    <javax.annotation-api>1.3.2</javax.annotation-api>
```

Would recommend to keep the property naming format and prefix all version related properties with `version`:
```suggestion
    <version.javax-annotation-api>1.3.2</version.javax-annotation-api>
```